vignettes/RPyGeo.Rmd

```diff
@@ -16,143 +21,199 @@ knitr::opts_chunk$set(
 )
 ```
 
-## Introduction
+# Introduction
 
-RPyGeo is a package providing access to ArcGIS geoprocessing tools in R. It archives this by building an interface between R and the ArcPy Python side-package via the reticulate package. 
+**RPyGeo** establishes an interface to the geoprocessing tools of ArcGIS from within R.
+Since ArcGIS only provides a Python API for a low-level access of its C++ based geoalgorithms, **RPyGeo** establishes a tunnel to Python via the **reticulate** package.
+This extends R's spatial capabilities [@bivand_applied_2013; @hijmans_raster_2017; @pebesma_simple_2018] by the geoalgorithms and the geoprocessing power of ArcGIS.
+Combining this with the statistical and data science power of R opens the way to advanced console-base statistical geoprocessing [@muenchow_rqgis_2017]. 
 
-RPyGeo is not just a wrapper package for ArcPy. It aims to create an environment, in which the user can perform ArcGIS and R functions on the same dataset in a contiguous workflow. It should not be necessary to touch Python or ArcGIS. 
+To use **RpyGeo** properly, at least a basic understanding of **ArcPy** is necessary.
+ArcPy is a Python side-package that allows geographic data analysis through ArcGIS from the Python command line.
+ArcPy is organized into modules which are Python files with functions and classes. The ArcPy main module `arcpy` (and `arcgis`, respectively depending on the used ArcGIS version <!-- Marc, pls check if this is right -->) provides (geographic) classes and several hundred functions.
+Further modules (e.g., `data access`) and extensions (e.g., `spatial analysis`) further extend the ArcPy main module.
```

Comment:
Fabian wrote that RPyGeo depends on ArcMap 10.0 at least. Since then it is called ArcPy module. But I think it is called ArcGIS module if you use ArcGIS API for Python, which is possible with RPyGeo. I do not have a source for that and I cannot test how it is called because I do not have access to ArcGIS API for Python.

Reply:
@fapola Do you know more about this?

Reply:
Hi @be-marc, as far as I remember it is called 'arcgis' module indeed if you're using the ArcGIS API for Python. See the [API reference](https://esri.github.io/arcgis-python-api/apidoc/html/)! I also think you just need an ArcGIS Online account to gain access to the ArcGIS Python API but I am not sure about this anymore and still confused about all these ESRI license models :smiley:. It would definitely be interesting to do more testing in this direction since you can also use the ArcGIS API for Python on Linux.

Reply:
Thank you. Yes, we should do more testing with ArcGIS Python API and also with different ArcMap and ArcGIS Pro versions.